specification/cognitiveservices/OpenAI.Inference/models/completions.create.tsp

```diff
@@ -189,14 +189,12 @@ model Choice {
   index: int32;
 
   #suppress "@azure-tools/typespec-azure-core/no-nullable" "The operation already returns nulls"
-  #suppress "@azure-tools/typespec-autorest/union-unsupported" "https://github.com/Azure/typespec-azure/issues/3180"
```

Comment:
One thing a little strange, is I was able to remove the `#suppress` for `CompletionsLogProbabilityModel`, but there is no change for this type in the autorest (only for `CompletionsFinishReason`).  Is this OK?

Reply:
I think that suppress was incorrect, there wasn't an error reported here. 

`CompletionsLogProbabilityModel` is a model

Reply:
I will try to remove this suppress in main

Reply:
Removing this in main passes CI, so it must have unnecessarily been added in error.

https://github.com/Azure/azure-rest-api-specs/pull/24562